src/analysis/deletion/removeZeroFluxRxns.m

```diff
@@ -0,0 +1,76 @@
+function modelOut = removeZeroFluxRxns(model, rxns)
+% Removes the reactions that cannot have any fluxes .
+% This function sets each reaction as the objective function and applies
+% FBA to find maximum and minuimum flux that can be carried through that
+% reaction, and removes the reactions that cannot have any fluxes.
+% notice: the results have functional dependency, not structural. results
+% may change due to a change in substrate for a model.
+%
+% USAGE:
+%    new_Model = removeZeroFluxRxns(model, reactions)
+%
+% INPUTS:
+%    model:                 COBRA model structure (must define `.rxns`, `.lb`, `.ub`)
+%
+%OPTIONAL INPUT
+%    reactions:             Cell array of reaction names in the model
+%
+% OUTPUT:
+%    new_model:             Curated model, the model with removed reactions
+%                           incapable of having a flux value, and corrected
+%                           upper and lower bounds.
+%
+% .. Author: - Farid Zare  5/31/2019
+
+if ~isstruct(model)
```

Comment:
If you test for some properties, why not for the existence of `model.S` which is necessary t run optimizeCbModel.
Alternatively, you could use `verifyModel(model,'FBAOnly',true)` to check all fields that are relevant for FBA calculations. The result is a struct that is empty if there are no errors and contains error information if there are any errors. Admittedly, a printout function for the error struct is currently still missing, but we can add that if you need it. If you just want to check a yes/no you could call: `verifyModel(model,'FBAOnly',true,'simpleCheck', true)` and if this is false e.g. exist with an error like:
"The input model is invalid. Please check the model with verifyModel(model) and correct all indicated errors"